### PR TITLE
chore(bigtable): more text proxy tune-up

### PR DESCRIPTION
**Pull request by telpirion**
This PR:

* Changes how table IDs are passed to `Client.Open()`
* Changes the error returned to the caller for a closed client.

### CODE REVIEW

File: bigtable/internal/testproxy/proxy.go [hunk 1]
```diff
@@ -507,15 +529,15 @@ type goTestProxyServer struct {
 
 // client retrieves a testClient from the clientIDs map. You must lock clientsLock before calling
 // this method.
```
Comment (quartzmo) [inline bigtable/internal/testproxy/proxy.go:531]: Maybe move this comment into an error message?

Comment (quartzmo) [reply bigtable/internal/testproxy/proxy.go:531]: > You must lock clientsLock before calling this method.

**Review by quartzmo — APPROVED** [review]
LGTM, but with question.

File: bigtable/internal/testproxy/proxy.go (hunk 1 above)
Comment (telpirion) [reply bigtable/internal/testproxy/proxy.go:531]: Ack. That makes sense, but is out-of-scope for this review. I'll consider your suggestion for future improvements.

**Pull request merged**